package.json

```diff
@@ -5,6 +5,8 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
+    "reconvert_data":"rm data/processed/* && cd scripts && python main.py",
```

Comment:
Will eventually want one command that handles all imports and cleanings but it is not this day